packages/compiler/lib/decorators.ts

```diff
@@ -16,6 +16,21 @@ export function $doc(program: Program, target: Type, text: string) {
   program.stateMap(docsKey).set(target, text);
 }
 
+export function $formatDoc(program: Program, target: Type, formatText: string, sourceObject: Type) {
```

Comment:
Should just the $doc support that?

Reply:
We can definitely discuss it.  I think if we want `doc` to support it, we should think a bit more about how to make it more flexible (accessing sub-properties, potentially using filters of some sort on the returned value)

Reply:
@bterlson @nguerrera Any thoughts on whether `$doc` should support the basic formatting approach I added to `$formatDoc` by default?

Reply:
And the usage would read nicer.